src/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link
+			rel="stylesheet"
+			href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0">
+		<title>ToDo List Project</title>
+	</head>
+	<body>
+		<header>
+			<div class="to-do-title flex-row border-bottom">
+				<h3>Today's To Do</h3>
+				<span class="material-symbols-outlined allClear"> autorenew </span>
+			</div>
+			<form class="to-do-add flex-row border-bottom">
+				<input type="text" class="to-do-input" placeholder="Add to your list...">
+				<button type="submit">
+					<span class="material-symbols-outlined to-do-addTask">add_task</span>
+				</button>
+			</form>
+		</header>
+		<main class="to-do-items"></main>
```

Comment:
please always use 2 inline spacing. it will make your code look clean and more organised also always add at least a section in the main tag as required in HTML best practice

Reply:
@eerapheal thanks for the comments